dotenv/README.md

```diff
@@ -22,17 +22,20 @@ This library loads environment variables from a _.env_ file. This is convenient
 
 ### Loading at runtime
 
-```rs
-use dotenvy::dotenv;
+```rust
 use std::env;
+use std::error::Error;
 
-fn main() {
-    // load environment variables from .env file
-    dotenv().expect(".env file not found");
```

Comment:
This line notably produced misleading error messages, claiming the file was not found when it actually contained invalid variable declarations.

Reply:
Good point